docs/options/external.md

```diff
@@ -0,0 +1,125 @@
+# external
+
+- **Type:** `string | RegExp | (string | RegExp)[] | ((id: string, parentId: string | undefined, isResolved: boolean) => NullValue<boolean>)`
+- **Optional:** Yes ✅
+
+Specifies which modules should be treated as external and not bundled. External modules will be left as import statements in the output.
+
+## Examples
+
+### String pattern
+
+```js
+export default {
+  external: 'react',
+};
+```
+
+### Regular expression
+
+```js
+export default {
+  external: /node_modules/,
```

Comment:
Can we give a better example
Of the node modules regex is being discouraged further below?